rotkehlchen/tests/api/test_custom_assets.py

```diff
@@ -425,24 +426,19 @@ def test_deleting_custom_assets(rotkehlchen_api_server, globaldb):
         ),
         json={'identifier': custom3_id},
     )
-    result = assert_proper_response_with_result(response)
-    assert result is True
+    assert_proper_response(response)
     assert globaldb.get_asset_data(identifier=custom3_id, form_with_incomplete_data=False) is None
 
-    # Try to delete custom1 but make sure it fails. It's used by custom2
+    # Delete custom1 and assert it works
```

Comment:
Yes that seems wrong. Is unexpected behaviour that if you deleted an asset referenced by this in forked/swapped_for you also delete the other one.

Reply:
Makes sense. I'll change the behaviour to set null on delete